tests/dvm/.gitignore

```diff
@@ -1,3 +1,3 @@
 node_modules/
 build/
-target/
+package.json
```

Comment:
I think this file can be removed.

There is .gitignore in the root which will handle these.

Reply:
Okay.